src/core/gui/PageView.cpp

```diff
@@ -624,6 +624,20 @@ auto XojPageView::onButtonReleaseEvent(const PositionInputData& pos) -> bool {
 }
 
 auto XojPageView::onKeyPressEvent(GdkEventKey* event) -> bool {
+    if (this->textEditor) {
+        if (this->textEditor->onKeyPressEvent(event)) {
+            return true;
+        }
+    } else if (this->inputHandler) {
+        if (this->inputHandler->onKeyEvent(event)) {
+            return true;
+        }
+    } else if (this->verticalSpace) {
+        if (this->verticalSpace->onKeyPressEvent(event)) {
+            return true;
+        }
+    }
+
```

Comment:
There may be unintended consequences due to the change in key handling behavior here, but I think these should be _good_ consequences, as some events that we've ignored in the past will now be forwarded to the appropriate places.

Reply:
(no need to take action on this)